andorigna/src/App.js

```diff
@@ -0,0 +1,32 @@
+
+import './App.css';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements
+} from "react-router-dom";
+import Home from './component/page/Home/Home';
+import Contact from './component/page/Contact/Contact';
+import About from './component/page/About Us/About';
+import Sign  from './component/page/Sign Out/Sign';
+import Layout from './component/page/Navbar/Layout';
+
+const router = createBrowserRouter(createRoutesFromElements(
+
+  <Route element ={<Layout/>}>
```

Comment:
There are simpler ways of creating routes with <BrowserRouter /> https://reactrouter.com/en/main/router-components/browser-router